src/couch_replicator_httpc.erl

```diff
@@ -98,6 +98,7 @@ send_ibrowse_req(#httpdb{headers = BaseHeaders} = HttpDb, Params) ->
         lists:ukeymerge(1, get_value(ibrowse_options, Params, []),
             HttpDb#httpdb.ibrowse_options)
     ],
+    backoff_before_request(Worker, HttpDb, Params),
```

Comment:
I remember that you mentioned backing off before the request was beneficial. Was there a logical reasoning behind this or did it seem to more performant based off tests?